monitor.go

```diff
@@ -59,9 +61,14 @@ type stageMonitor struct {
 "isDiscrete": true,
 "visible": true
 */
-func newStageMonitor(g reflect.Value, v specsp) (*stageMonitor, error) {
+func newMonitorWidget(g reflect.Value, v specsp) (*MonitorWidget, error) {
```

Comment:
It should be ok to omit `color` in config (`v` here), as it makes no sense for current monitor style. While it panics if I do so

```
panic: color is nil
```